sdk/communication/azure-communication-email/src/test/java/com/azure/communication/email/EmailTestBase.java

```diff
@@ -71,4 +76,15 @@ private static TestMode initializeTestMode() {
             return TestMode.PLAYBACK;
         }
     }
+
+    static Stream<Arguments> getTestParameters() {
+        // When this issues is closed, the newer version of junit will have better support for cartesian product of
+        // arguments - https://github.com/junit-team/junit5/issues/1427
+        List<Arguments> argumentsList = new ArrayList<>();
+
+        getHttpClients()
+            .forEach(httpClient -> Arguments.of(httpClient));
```

Comment:
The client is not added to the `argumentsList` here. So, the argumentsList is always empty.